src/app/shared/model/feeder-podcast.model.ts

```diff
@@ -120,6 +134,12 @@ export class FeederPodcastModel extends BaseModel {
     } else {
       data.itunesCategories = [];
     }
+
+    data.explicit = this.explicit || null;
```

Comment:
same comment on name/email, but since the debate we had was specifically about explicit commenting here- don't 🔫 the messenger, but @cavis has told me in the past that we set empty values to empty string and not to use `null`